apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -43,15 +43,16 @@
   void enqueue(@Nullable Callback<T> callback);
 
   /**
-   * Sets the {@link CacheControl} strategy for an ApolloCall object.
+   * Sets the {@link ResponseFetcher} strategy for an ApolloCall object.
    *
-   * @param cacheControl the CacheControl strategy to set
+   * @param fetcher the {@link ResponseFetcher} to use.
    * @return The ApolloCall object with the provided CacheControl strategy
```

Comment:
`@return The ApolloCall object with the provided CacheControl strategy` -> 
`@return The ApolloCall object with the provided fetch strategy`